packages/clerk-js/src/core/resources/SignUp.ts

```diff
@@ -81,54 +81,25 @@ export class SignUp extends BaseResource implements SignUpResource {
     this.fromJSON(data);
   }
 
-  create = async (params: SignUpCreateParams): Promise<SignUpResource> => {
-    const paramsWithCaptcha: Record<string, unknown> = params;
-
-    if (!__BUILD_DISABLE_RHC__) {
-      const {
-        captchaSiteKey,
-        canUseCaptcha,
-        captchaURL,
-        captchaWidgetType,
-        captchaProvider,
-        captchaPublicKeyInvisible,
-      } = retrieveCaptchaInfo(SignUp.clerk);
-
-      if (
-        !this.shouldBypassCaptchaForAttempt(params) &&
-        canUseCaptcha &&
-        captchaSiteKey &&
-        captchaURL &&
-        captchaPublicKeyInvisible
-      ) {
-        try {
-          const captchaParams = await getCaptchaToken({
-            siteKey: captchaSiteKey,
-            widgetType: captchaWidgetType,
-            invisibleSiteKey: captchaPublicKeyInvisible,
-            scriptUrl: captchaURL,
-            captchaProvider,
-          });
-
-          paramsWithCaptcha.captchaToken = captchaParams.captchaToken;
-          paramsWithCaptcha.captchaWidgetType = captchaParams.captchaWidgetType;
-        } catch (e) {
-          if (e.captchaError) {
-            paramsWithCaptcha.captchaError = e.captchaError;
-          } else {
-            throw new ClerkRuntimeError(e.message, { code: 'captcha_unavailable' });
-          }
-        }
+  create = async (_params: SignUpCreateParams): Promise<SignUpResource> => {
+    let params: Record<string, unknown> = _params;
+
+    if (!__BUILD_DISABLE_RHC__ && !this.clientBypass() && !this.shouldBypassCaptchaForAttempt(params)) {
+      const captchaChallenge = new CaptchaChallenge(SignUp.clerk);
+      const captchaParams = await captchaChallenge.managed();
+      if (!captchaParams) {
+        throw new ClerkRuntimeError('', { code: 'captcha_unavailable' });
```

Comment:
🤔  Is this error getting set to the Card component? If yes, then we show no message now, while before we had the `e.message` here.

Reply:
Won't this get the message from the localization file instead? 
We don't know what e.message was and we don't want to show CF errors to the end users for sure.

Reply:
Yes, we have this code on the localization file

Reply:
Thanks!